src/versions.rs

```diff
@@ -10,11 +10,14 @@ pub const HC_SPIN_VERSION: &str = "^0.400.0-dev.3";
 /// npm: <https://www.npmjs.com/package/@holo-host/web-sdk>
 pub const WEB_SDK_VERSION: &str = "^0.6.20-prerelease";
 
-pub const HDI_VERSION: &str = holochain::HDI_VERSION;
+/// crates.io <https://crates.io/crates/hdi/versions>
+pub const HDI_VERSION: &str = "0.6.0-dev.3";
```

Comment:
Does any documentation about how to bump the scaffolding to a new Holochain version need to change?

Reply:
Good catch, updated the maintenance docs